cmd/upload.go

```diff
@@ -85,6 +86,11 @@ func NewUploadCommand(options *UploadOptions, metadataProviders []metadata.Provi
 
 			timing := util.NewTiming("total")
 
+			var mapsetId *uint64 = nil
```

Comment:
Why is `mapsetId` a pointer? Can't it just be a `uint64`? Also, since the zero value of a unit64 isn't a legal mapset id, could you use that to indicate an unset id, rather than `nil`?

Reply:
It's more of a style preference. I've changed it to use 0.